src/content/en/2021/media.md

```diff
@@ -1,20 +1,900 @@
+
+#### Decoding
+
+The `decoding` attribute on `<img>` serves as a useful point of contrast to highlight native lazy loading's success. <a hreflang="en" href="https://www.chromestatus.com/feature/4897260684967936">First supported</a> in 2018–about a year before native lazy-loading–the `decoding` attribute allows developers to prevent large image decode operations from blocking the main thread. It provides functionality that not all web developers need or understand, and that shows in the usage data. <a hreflang="en" href="https://docs.google.com/spreadsheets/d/1nwkpviC3gNhRb48i8OoIgtJx1ckqRjJGW7uc7Gdi_sI/edit?pli=1#gid=1934121343">`decoding` is used on just 1% of pages, and on only 0.3% of `<img>` elements</a>.
+
+#### Accessibility
+
+When you embed contentful images on webpages, you should make their content as accessible as possible for non-visual users. I note this only to [defer you to the Accessibility chapter](./accessibility#images), whose in-depth analysis of image accessibility on the web found small year-over-year progress in this area, but mostly: a whole lot of room for improvement.
+
+#### Responsive images
+
+In 2013, a suite of features enabling adaptive image loading on responsive websites landed, to much fanfare. Eight years in, how are responsive image features being used?
+
+##### Srcset
+
+First, let us consider the [`srcset` attribute](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset), which allows developers to supply multiple possible resources for the same `<img>`:
+
+###### Feature adoption
+
+{{ figure_markup(
+  caption="Percent of pages that use `srcset`.",
+  content="31%",
+  classes="big-number",
+  sheets_gid="1594311632",
+  sql_file="image_srcset_sizes.sql"
+)
+}}
+
+{{ figure_markup(
+  image="srcset-descriptor-adoption.png",
+  caption="`srcset` descriptor adoption.",
+  description="A bar chart showing... TODO",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=89579817&format=interactive",
+  sheets_gid="1030564653",
+  sql_file="TODO.sql"
+  )
+}}
+
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th></th>
+        <th>Percent of pages using feature</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th>`x` descriptors</th>
+        <td class="numeric">6%</td>
+      </tr>
+      <tr>
+        <th>`w` descriptors</th>
+        <td class="numeric">24%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="`srcset` descriptor adoption.", sheets_gid="1030564653", sql_file="image_srcset_descriptor.sql") }}</figcaption>
+</figure>
+
+{# TODO I was wondering if this was desktop or mobile, then discovered you had a graph which showed both, so repalced this table with that. I see the table is back now. Is it needed now we have the graph which In think is more interesting? #}
```

Comment:
What do you think @eeeps ?

Also we have a big figure, a figure, and then a table. While we like to be stats driven that's three figures in a row before we get to any text. If we are keeping all three should we add some text in between?

Reply:
What it looks like here: https://20211207t160821-dot-webalmanac.uk.r.appspot.com/en/2021/media#feature-adoption

Reply:
The chart and the table are duplicitous. Deleting the table and adding a sentance to break up the back-to-back figures.

Reply:
Done.